### PR TITLE
Add GNOME

### PR DESCRIPTION
**Pull request by Kazhnuz**
**https://github.com/Kazhnuz/awesome-gnome**

**This is a list of applications, extensions, modules and tools for the GNOME Desktop, to help people that use it to find more easily first- and third-party apps and add-on, or to create their own.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- It's the result of hard work and the best I could possibly produce.
	- Non-generated Markdown file in a GitHub repo.
	- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- Not a duplicate.
	- Only has awesome items. Awesome lists are curations of the best, not everything.
	- Includes a project logo/illustration whenever possible.
		- Placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
		- Should be placed on the right side of the readme heading.
		- Should link back to this list.
	- Has a Table of Contents section.
		- Should be named `Contents`, not `Table of Contents`.
		- Should be the first section in the list.
		- Should only have one level of sub-lists, preferably none.
	- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- If you use a license badge, it should be SVG, not PNG.
	- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- Each link description starts with an uppercase character and ends with a period.<br>Example: `- [AVA](…) - JavaScript test runner.`
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Go to the top and read it again.

### CODE REVIEW

**Comment by davisonio** [comment]
Nice list! LGTM 👍

File: readme.md [hunk 1]
```diff
@@ -86,6 +86,7 @@
 - [WebExtensions](https://github.com/bfred-it/Awesome-WebExtensions) - Cross-browser extension system.
 - [RubyMotion](https://github.com/motion-open-source/awesome-rubymotion) - Write cross-platform native apps for iOS, Android, macOS, tvOS, and watchOS in Ruby.
 - [Smart TV](https://github.com/vitalets/awesome-smart-tv) - Create apps for different TV platforms.
+- [GNOME](https://github.com/Kazhnuz/awesome-gnome) - Simple and distraction-free desktop environnement for Linux
```
Comment (davisonio) [inline readme.md]: * environment

also would add full stop at the end.

Comment (Kazhnuz) [inline readme.md]: Sorry for the typo.

Both are fixed, thank you ! :)

**Comment by sindresorhus** [comment]
Nice work! :)

I'd also like to see some links to community resources: Stack Overflow tag, IRC, chat, forums, communities, Reddit, etc. Example: https://github.com/sindresorhus/awesome-electron#community

**Comment by Kazhnuz** [comment]
Thank you! :)

I've added the section : https://github.com/Kazhnuz/awesome-gnome#community

**Comment by aleksandar-todorovic** [comment]
This looks awesome!

The only complaint I have is this one:

> Items marked with <GNOME_ICON> are applications officially maintained and distributed by the GNOME Project as part of the core set of applications and tools.

Shouldn't some more resources (like GNOME Games and Tweak Tool) have that icon? Not sure where you pull the "core set of applications" from. The only similar criteria for that seems to be [this Wikipedia article](https://en.wikipedia.org/wiki/GNOME_Core_Applications) and both of the applications I've mentioned have been included in that article.

**Comment by OriginalFannibal** [comment]
Hey, I am a contributor to this list.

Here is the present list of GNOME core applications: [GNOME 3.26 Core Applications](https://blogs.gnome.org/mcatanzaro/2017/08/13/gnome-3-26-core-applications/).

This is the list that we use as a criteria to mark an app as a core GNOME app. As you can see, neither GNOME Games or Tweak Tool are on that list.

**Comment by aleksandar-todorovic** [comment]
Ah, so from the official set of GNOME applications issued with the latest version. Makes sense! Well, since @sindresorhus's suggestion was already implemented, I don't see a reason not to have this list included.

**Pull request merged**